### DISCUSSION TITLE
Issue with links in Email Templates

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15619] [Feature improvement]
Trying to set clickable link in Email Template but when I pasting field variable in URL field (Link dialog in editor) after saving this variable is cleared. Way around? Fix?

**Comment by rskrzypczak** [comment]
Add link domain to:
https://github.com/YetiForceCompany/YetiForceCRM/blob/9970736daefa79ea1cd98471041517a1960d20a0/config/Security.php#L181-L182

or set it to **false**, to turn off the verification globally.

**Comment by s2b-git** [comment]
Just gave up creating button in workflow email task. Does anyone have advice on what method use? Purifier is off.

:-)
